starters/vue3-xstate-css/src/router/index.ts

```diff
@@ -0,0 +1,25 @@
+import { createRouter, createWebHistory } from 'vue-router';
+import HomeView from '../views/HomeView.vue';
+
+const router = createRouter({
+	history: createWebHistory(import.meta.env.BASE_URL),
+	routes: [
+		{
+			path: '/',
+			name: 'home',
+			component: HomeView,
```

Comment:
Lol I had this dynamic before, but Simone suggested it should be a direct import since the home page is basically always expected to exist. 
https://github.com/thisdot/starter.dev/pull/1208#discussion_r1185066650

Reply:
Alright, after a bit of discussion and looking at the official Vue Router docs, I'm going to revert this and use the dynamic import. 

For reference if anyone is curious, in the [Vue Router docs](https://router.vuejs.org/guide/advanced/lazy-loading.html#lazy-loading-routes) they say "In general, it's a good idea to always use dynamic imports for all your routes." 

And per some internal discussion, if the root page is visited the component will need to be imported whether it's static or dynamic. But if someone uses this kit to make their own project, and they end up having other pages that are visited directly (ie a link to a blog post or something), it's feasible that some users won't actually need the home component loaded.